src/order_cmd.cpp

```diff
@@ -1881,7 +1881,11 @@ uint16_t GetServiceIntervalClamped(int interval, bool ispercent)
  */
 static bool CheckForValidOrders(const Vehicle *v)
 {
-	return std::ranges::any_of(v->Orders(), [](const Order &order) { return order.IsGotoOrder(); });
+	/* Check if vehicle has any valid orders
+	Function is only called for aircraft, no type check needed*/
```

Comment:
Comment formatting isn't quite right, you need another * at the start of the second line, a space before the comment closing tag, and periods on both sentences.

```suggestion
	/* Check if vehicle has any valid orders.
	 * Function is only called for aircraft, no type check needed. */
```